# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../adgen/library-relocation.patch
# Copyright (C) 2007 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

diff -u -r adgen.orig/adgen.py adgen/adgen.py
--- adgen.orig/adgen.py	2007-05-02 23:03:00.000000000 +0200
+++ adgen/adgen.py	2007-06-17 13:11:21.000000000 +0200
@@ -29,6 +29,7 @@
 import os
 import shutil
 import sys
+sys.path.append("$LIBDIR")
 
 import util.adgenDebug as adgenDebug
 import util.adgenI18N as adgenI18N
@@ -616,4 +617,4 @@
     pass
     
 if __name__ == "__main__":
-    BuildWebSite()
\ No newline at end of file
+    BuildWebSite()
